test/__snapshots__/Autocomplete.test.tsx.snap

```diff
@@ -50,7 +50,7 @@ Object {
               >
                 <button
                   aria-label="Clear"
-                  class="MuiButtonBase-root-91 MuiIconButton-root-82 MuiAutocomplete-clearIndicator-12"
+                  class="MuiButtonBase-root-91 MuiIconButton-root-82 MuiAutocomplete-clearIndicator-12 MuiAutocomplete-clearIndicatorDirty-13"
```

Comment:
I had to regenerate the snapshot for this. I assume that MUI changed their syntax here.

Reply:
This is correct. This is the bug I introduced. I couldn't figure out why this snapshot was failing, so I (incorrectly) passed it. Stupid me. Sorry!